### PR TITLE
all files in src/wp-admin related to issue #145

### PR DESCRIPTION
**Pull request by senlin**
all files in src/wp-admin related to issue ClassicPress/ClassicPress-v1#203: Clean up references to WP ticket numbers

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wp-admin/css/color-picker.css [hunk 1]
```diff
@@ -49,7 +49,7 @@
 }
 
 .wp-color-result:active {
-	/* See Trac ticket #39662 */
+	/* See Trac ticket https://core.trac.wordpress.org/ticket/39662 */
```
Comment (nylen) [inline src/wp-admin/css/color-picker.css]: Might as well remove "Trac ticket" here too.

File: src/wp-admin/css/common.css [hunk 2]
```diff
@@ -2839,7 +2839,7 @@ div.action-links {
 	font-size: 13px;
 }
 
-/* For non-js plugin installation screen ticket #36430. */
+/* For non-js plugin installation screen ticket https://core.trac.wordpress.org/ticket/36430. */
```
Comment (nylen) [inline src/wp-admin/css/common.css]: Might as well remove "ticket" here too.

File: src/wp-admin/js/customize-controls.js [hunk 3]
```diff
@@ -6320,7 +6320,7 @@
 				title: api.l10n.previewIframeTitle,
 				name: 'customize-' + previewFrame.channel()
 			} );
-			previewFrame.iframe.attr( 'onmousewheel', '' ); // Workaround for Safari bug. See WP Trac #38149.
+			previewFrame.iframe.attr( 'onmousewheel', '' ); // Workaround for Safari bug. See WP Trac https://core.trac.wordpress.org/ticket/38149.
```
Comment (nylen) [inline src/wp-admin/js/customize-controls.js]: Can remove "WP Trac" here.

**Comment by nylen** [comment]
I didn't comment on all instances, but I think anywhere it says "Trac ticket" or "ticket" or "WP Trac" we can remove that wording too.

**Comment by nylen** [comment]
I didn't comment on all instances, but I think anywhere it says "Trac ticket" or "ticket" or "WP Trac" we can remove that wording too.

**Comment by senlin** [comment]
I managed to find a few more :)

**Pull request merged**